google-cloud-clients/google-cloud-bigtable/src/test/java/com/google/cloud/bigtable/data/v2/stub/EnhancedBigtableStubSettingsTest.java

```diff
@@ -418,7 +420,7 @@ public void bulkMutateRowsSettingsAreNotLostTest() {
     assertThat(builder.bulkMutateRowsSettings().getRetryableCodes())
         .containsAtLeast(Code.ABORTED, Code.DEADLINE_EXCEEDED);
     assertThat(builder.bulkMutateRowsSettings().getRetrySettings()).isEqualTo(retrySettings);
-    assertThat(builder.bulkMutateRowsSettings().getBatchingSettings())
+    assertThat(builder.bulkMutateRowsSettings().build().getBatchingSettings())
```

Comment:
why is this change necessary?

Reply:
We don't have getters anymore in new BatchingCallSettings.Builder. So I had an option either to review this assertion or build and get the BatchingSettings.

Reply:
Did you mean add a getter to `BatchingCallSettings.Builder`?

Reply:
yep